tuxemon/save_upgrader.py

```diff
@@ -84,6 +84,12 @@ def upgrade_save(save_data: Dict[str, Any]) -> SaveData:
     save_data["battle_history"] = save_data.get("battle_history", {})
     save_data["money"] = save_data.get("money", {})
 
+    # set money old savegame and avoid getting the starter
+    if len(save_data["money"]) == 0:
```

Comment:
`if not save_data["money"]`

Is more idiomatic than

`if len(save_data["money"]) == 0`

Reply:
Thanks, fixed.